src/scraper/stockx_guide.ts

```diff
@@ -0,0 +1,366 @@
+        //await DoneStr.push(e.url);
+      } else if ("Collect") {
+        //await CollectStr.push(e.url);
+      } else if ("Error") {
+        //
+      }
+    });
+
+    /*
+       await c.map( async (e)=>{
+         this.collectProductsInPage(e.url)
+       })*/
+    /*
+     await c.map( async (e)=>{
+       setTimeout(async ()=>{ await this.collectProductsInPage(e.url)}, 1000)
+     })*/
+  };
+
+  createUrl = async (baseURL: string, querys: Object): Promise<string> => {
+    Object.keys(querys).forEach((e: string) => {
+      baseURL += `&${e}=${querys[e]}`;
+    });
+    return baseURL;
+  };
+
+  collectAll = async (): Promise<Array<string>> => {
+    const _tags: Array<string> = [];
+    const urls: Array<string> = [];
+    await brands.map((e: { name: string; models: Array<string> }) => {
+      e.models.map((v: string) => {
+        if (e.name == v) _tags.push(e.name);
+        else _tags.push(e.name + "," + v);
+      });
+    });
+    await _tags.map(async (e: string) => {
+      years.map(async (year: string) => {
+        const url = await this.createUrl(ModelBaseURL, { _tags: e, year });
+        urls.push(url);
+      });
+    });
+    return urls;
+  };
+
+  checkProductURL = async (baseUrl: string): Promise<any> => {
+    let newURL: any = await prisma.uRLs({ where: { url: baseUrl } });
+    const response: any = await axios
+      .get(baseUrl, headers)
+      .catch(error =>
+        console.log("\n\n[ERROR] Function checkProductURL\n", baseUrl, "\n")
+      );
+
+    if (!response || !response.data) {
+      console.log("\n[ERROR] In Function createProductURL\n");
+      return "Error";
+    }
+    if (
+      newURL.length > 0 &&
+      newURL[0].ProductAmount == response.data.Pagination.total
+    ) {
+      const checking: any = await prisma.products({
+        where: { urlForCheck: baseUrl }
+      });
+      if (newURL[0].ProductAmount > checking.length) {
+        await console.log("[Warning] There are unregistered data in ", baseUrl);
+        await prisma.updateURL({
+          data: { isComplete: false },
+          where: { id: newURL[0].id }
+        });
+        return "Collect";
+      } else if (newURL[0].ProductAmount < checking.length) {
+        await console.log("[Warning] There are too much data in ", baseUrl);
+        await prisma.updateURL({
+          data: { isComplete: false },
+          where: { id: newURL[0].id }
+        });
+        return "Collect";
+      } else {
+        await console.log("[Check] There isn't unregistered data in ", baseUrl);
+        await prisma.updateURL({
+          data: { isComplete: true },
+          where: { id: newURL[0].id }
+        });
+        return "Done";
+      }
+      return;
+    }
+    if (newURL.length == 0) {
+      const { total, lastPage } = response.data.Pagination;
+      newURL = await prisma.createURL({
+        url: baseUrl,
+        ProductAmount: total,
+        lastPage,
+        isComplete: false
+      });
+      await console.log(
```

Comment:
console.log는 await이 필요한 코드가 아닙니다.
```suggestion
      console.log(
```